sdk/python/feast/feature_store.py

```diff
@@ -358,7 +360,7 @@ def _get_features(self, features: Union[List[str], FeatureService],) -> List[str
         return _feature_refs
 
     @log_exceptions_and_usage
-    def apply(
+    def plan(
```

Comment:
it's extremely unintuitive to me that `plan` already has a concept of objects to keep and delete - after all, the point of planning should be to determine which things to keep, add, and delete

what I had in mind with plan was to reconstruct a full RegistryProto from a ParsedRepo, and then compare that with the existing registry - thoughts on this approach vs. yours?

Reply:
Okay, I think that make sense. Lemme update.